### PR TITLE
Brooming up

### PR DESCRIPTION
**Pull request by federicomarini**
Prepping up the required content and looks before the upcoming Bioc submission

### CODE REVIEW

File: DESCRIPTION [hunk 1]
```diff
@@ -24,11 +24,13 @@ Authors@R:
...
            comment = c(ORCID = '0000-0002-3564-4813')))
-Description: Contains a set of modes for iSEE, allowing easy configuration of 
-    iSEE applications.
+Description: iSEEu (the iSEE universe) contains diverse functionality to extend 
+    the usage of the iSEE package, including additional classes for the panels, 
+    or modes allowing easy configuration of iSEE applications.
 License: MIT + file LICENSE
 Encoding: UTF-8
 Depends: 
+    R (>= 4.0.0),
```
Comment (LTLA) [inline DESCRIPTION]: Fix this.

Comment (federicomarini) [reply DESCRIPTION]: Just to be removed then?
If I recall correctly, sometimes BiocCheck complains to have it specified.

Comment (LTLA) [reply DESCRIPTION]: Only if you have serialized objects.

File: README.md [hunk 2]
```diff
@@ -1,14 +1,103 @@
...
+The `iSEEu` package contains material and code that extends the `iSEE` package (https://github.com/iSEE/iSEE).
+
+We welcome contributions from the community, see below for more instructions. 
+For example, during the Developer Day at the European Bioconductor 2019 conference (`#EuroBioc2019`, at the UCLouvain, in Brussels, Belgium), we proposed a hackathon-like session, and we focused on the design of "modes", i.e. preconfigured sets of panels and linked content to be used as starting setup when launching `iSEE`.
+
+## Installation
+
+`iSEEu` can be easily installed from Bioconductor using `BiocManager::install()`:
+
+```
```
Comment (LTLA) [inline README.md]: Should have `r` to indicate R syntax highlighting.

File: DESCRIPTION (hunk 1 above)
Comment (federicomarini) [reply DESCRIPTION]: Ok, gone then ;)

**Pull request merged**